github-actions/src/main/scala/org/typelevel/sbt/gha/GenerativePlugin.scala

```diff
@@ -348,8 +355,9 @@ ${indent(rendered.mkString("\n"), 1)}"""
       commands: List[String],
       params: Map[String, String],
       renderedShell: String,
-      renderedWorkingDirectory: String) =
-    renderedShell + renderedWorkingDirectory + "run: " + wrap(
+      renderedWorkingDirectory: String,
+      renderedContinueOnError: String) =
```

Comment:
To fix bincompat, add a deprecated overload of this method with the old signature that passes an empty string to `renderedContinueOnError`